napi_basics.md

```diff
@@ -0,0 +1,139 @@
+# N-API - Basics
+
+N-API is an API exposed by Node to developers that allow them to author Native 
+add-ons in an ABI stable manner. This means that these add-ons can work 
+against future versions of Node without recompilation, and developers get the 
+additional benefit that their add-ons can work independent of the underlying 
+JavaScript Virtual Machine being used by Node.
+
+APIs exposed in the N-API surface are generally used to create and manipulate
+JavaScript objects. Concepts and operations generally map to ideas specified 
+in the ECMA262 Language Specification. The APIs generally have the following 
+properties:
+- All N-API APIs generally return a status code of type `napi_status`. This
```

Comment:
I would prefer "N-API calls" or "N-API members". Perhaps it's easiest to just say "All APIs generally ..." since the whole document is about N-API.